Osiris/Hacks/Misc.cpp

```diff
@@ -506,3 +506,62 @@ void Misc::killMessage(GameEvent& event) noexcept
     cmd += "\"";
     interfaces.engine->clientCmdUnrestricted(cmd.c_str());
 }
+
+void Misc::drawBombDamage() noexcept
+{
+    const auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
```

Comment:
no need to define localplayer if config.misc.bombDamage is false

Reply:
Fixed.